src/prog_translator.h

```diff
@@ -62,7 +65,7 @@ class ProgTranslator {
   /// @brief translates solution from iface back into graph matches
   void FromProg();
 
-  const ProgTranslator::Context& ctx() const { return ctx_; }
+  const Context& ctx() const { return ctx_; }
```

Comment:
Well, this changed the API by moving replacing a struct with a typedef. structs, like classes, have constructors and destructors implicitly added to them, if they are not given.  These member functions are then added to the symbol table.  Because this struct is nested inside of a class, the structs members become nested too.  The typedef just renames the type and - though it does add the new name to the symbol table - it doesn't do anything else since the original symbols live with their original name.